doc/tae.rst

```diff
@@ -22,6 +22,10 @@ provide slightly different interfaces. You can see `SVM-example
       <algo> <instance> <instance specific> <cutoff time> <runlength> <seed> <algorithm parameters>
       python algo.py 123 0 10 25 12345 -param1 value1 -param2 value2 [...]
 
+  The first two parameters after the ``algo.py`` hold the instance name on which the algorithm is evaluated and  extra information about the instance (rarely used).
```

Comment:
how about `are the instance name, on which the target algorithm is evaluated, and extra information about the instance (rarely used).